sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/CHANGELOG.md

```diff
@@ -1,14 +1,16 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2025-05-05)
 
 ### Features Added
 
-### Breaking Changes
+- Support Change Plan
+- Create Neon Project during Neon Organization
+- Create Neon Project from Overview
+- Create Neon Branch
+- Get Neon Database Connection URI
+- List Neon Projects, Branches, Databases, Roles, Endpoints
```

Comment:
could we also include a line about the api-version upgrade?

Reply:
Done